data/json/snippets/zombie_anatomy.json

```diff
@@ -171,6 +171,72 @@
       "You cut into its abdomen, and discover its utter lack of internal organs, the expected cavity replaced by nothing but more flesh and muscle."
     ]
   },
+  {
+    "type": "snippet",
+    "category": "<zombie_harvest_grabber_general>",
+    "text": [
+      "The sheer remaining tension in each of the muscles of its arms and shoulders makes them unwind with a sonorous thwang each time you cut one of the supporting bone.",
+      "The tremendous growth of its upper body's muscle mass must have rendered the creature almost incapable of efficiently surveying its surroundings.",
+      "You realize the muscles of the zombie's arms are hollow, their marrow replaced by a thick oily black liquid. You wonder if the ridiculous muscle mass surrounding them is the other half of a strange hydraulic system.",
+      "You cut into the arms of the creature and realize that instead of what you would expect from what you know of normal human anatomy, its muscles wind around the bones in what looks like a tight spring..",
+      "While you are cutting into it, the creature suddenly moves and grabs one of your limbs. Despite considerable effort, you are unable to pry open the creature's iron grip and resign yourself to chop off the hand instead."
+    ]
+  },
+  {
+    "type": "snippet",
+    "category": "<zombie_harvest_dog_general>",
+    "text": [
+      "Opening its jaw wide, you notice the presence of multiple additional tongues deeper it in its throat.",
+      "You notice cysts in its fur, cutting into them causes an impressive amount of putrid black liquid to flow out.",
```

Comment:
Random cysts feel a bit close to mutant mess, maybe reference some later evo with them? Something line bone flakes or acid would connect them to the later morphs and show that evolution is a constant process not the flips we show ingame.